libs/zzz/solver/src/calc.ts

```diff
@@ -51,7 +53,11 @@ export function passSetFilter(
 /**
  * sum up stats from base + discs + 2p effects
  */
-export function applyCalc(baseStats: Stats, discs: DiscStats[]) {
+export function applyCalc(
+  baseStats: Stats,
+  conditionals: Partial<Record<CondKey, number>>,
+  discs: DiscStats[]
+) {
   const sum = { ...baseStats }
   const s = (key: string) => sum[key] || 0
 
```

Comment:
I think we can remove the `objSumInPlace(sum, setCount)` now, as the 4p constraint checking is done outside of the calc step